src/commands/apps-android-sha-create.ts

```diff
@@ -0,0 +1,56 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { AppAndroidShaData, createAppAndroidSha, ShaCertificateType } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+async function initiateAppAndroidShaCreation(
+  projectId: string,
+  appId: string,
+  options: { shaHash: string; certType: string }
+): Promise<AppAndroidShaData> {
+  const spinner = ora("Creating Android SHA certificate").start();
+  let certificateData;
+  try {
+    certificateData = await createAppAndroidSha(projectId, appId, options);
+    spinner.succeed();
+  } catch (err) {
+    spinner.fail();
+    throw err;
+  }
+
+  return certificateData;
+}
+
+function getCertHashType(shaHash: string): string {
+  shaHash = shaHash.replace(/:/g, "");
+  const shaHashCount = shaHash.length;
+  if (shaHashCount == 40) return ShaCertificateType.SHA_1.toString();
+  if (shaHashCount == 64) return ShaCertificateType.SHA_256.toString();
+  return ShaCertificateType.SHA_CERTIFICATE_TYPE_UNSPECIFIED.toString();
+}
+
+module.exports = new Command("apps:android:sha:create <appId> <shaHash>")
+  .description("add a SHA certificate hash for a given app id.")
+  .before(requireAuth)
+  .action(
+    async (appId: string = "", shaHash: string = "", options: any): Promise<AppAndroidShaData> => {
+      const projectId = getProjectId(options);
+
+      logger.info(
```

Comment:
This log message seems to say most of the same thing that the spinner says.  Maybe we should just put more of this information into the spinner message and remove this log message?

Reply:
ok removing, lets put
`Creating Android SHA certificate FA:CE:CC:44:9D:C8:01:63:93:4D:AB:06:FC:2E:20:D1:E6:5F:E1:9C:96:A3:51:38:85:B1:5B:DD:BE:26:54:DE with Android app Id 1:696982949839:android:e35b690950338b48f64cba`
or
`Creating Android SHA certificate FA:CE:CC:44:9D:C8:01:63:93:4D:AB:06:FC:2E:20:D1:E6:5F:E1:9C:96:A3:51:38:85:B1:5B:DD:BE:26:54:DE`
or
`Creating Android SHA certificate`

Reply:
The first one looks best to me!

Reply:
done